initPTB.m

```diff
@@ -1,61 +1,45 @@
 function [cfg] = initPTB(cfg)
 % This will initialize PsychToolBox
 % - screen
-%   - the windon opened takes the whole screen by default
-%   - set in debug mode with window transparency if necessary
-%   - can skip synch test if you ask for it (nicely)
-%   - gets the flip interval
-%   - computes the pixel per degree of visual angle
-%   - set font details
+% - the windon opened takes the whole screen unless
+% cfg.testingSmallScreen is set to true
+% - debug mode : skips synch test and warnings
+% - window transparency enabled by cfg.testingTranspScreen set to true
+% - gets the flip interval
+% - computes the pixel per degree of visual angle:
+% the computation for ppd assumes the windows takes the whole screenDistance
+% - set font details
 % - keyboard
+% - hides cursor
 % - sound
 
```

Comment:
what do you think about adding here the list of the output of visual/auditory params set by this function that are useful during the experiment? See below:

- cfg.screen : screen numbers where drawing the stimulation (external screen if available)
- cfg.win : window opened by PTB
- cfg.winRect : window rectangular in pixel coordinates  
- cfg.winWidth : window width in pixels
- cfg.winHeight : window height in pixels
- cfg.center : coordinate of the window center
- cfg.ppd : pixels per degree assuming the window fills the whole screen
- cfg.ifi : estimate of the monitor flip interval 
- cfg.monRefresh : monitor refresh rate
- cfg.vbl : (I don't think this output is useful)

Reply:
related to these cfg.ifi and cfg.vbl : are they useful if experimenter wants to check the monitor delays? I wanted to test some hardware issues (adapters + which laptop to use). and was thinking flip interval could be a parameter to compare the hardwares.

Reply:
Added some of them (and also mentioned the default values chosen for some of them) but there is clearly some room for improvement

Reply:
This is a good question that tickles a long-standing curiosity. If you want we can dig, I think you can have this info from some PTB's messages and we can add a miniguide to it (e.g. the simplest would be a link in the readme referencing the specific page in the PTB website that, IMHO, is not easy to navigate). I'll open an issue on that. 

@fedefalag might be expert here on that ;)